addons/website_slides/tests/test_slide_utils.py

```diff
@@ -128,3 +129,17 @@ def test_youtube_urls(self):
                     document = self.env['slide.slide']._find_document_data_from_url(url)
                     self.assertEqual(document[0], 'youtube')
                     self.assertEqual(document[1], id)
+
+@odoo.tests.tagged('archive_course')
+class TestUnpublish(slides_common.SlidesCase):
+
+    def test_archive(self):
+        # Archive a course
+        self.channel.toggle_active();
+        self.assertFalse(self.channel.active)
+        self.assertFalse(self.channel.is_published)
```

Comment:
Instead of a leading python comment, I prefer using the second arguments of the assert:
```py
self.assertFalse(self.channel.active, "Active should be False after archiving the channel")
self.assertFalse(self.channel.active, "Published should be set to False automatically when archiving the channel")
...
```

Same thing for here below (you get the idea).

It will log this string when the test fails, which helps to understand what's going on.

Reply:
I don't really agree with strings which are generally noise (and not maintained when updated). I think obvious action do not require additional strings. For example toggle_active + check channel.active is False is obvious to me, no need to write a sentence that may be more confusing than reading the code.

When reading the test I think only "functional" points should be explained (like "published should be de-activated when archiving", this makes sense).

Reply:
This is the usual debate between commenting code vs self documenting code. Both are valid point of view.

But from what I have seen inside the Odoo code base, self documenting code seams to be the norm.
I have updated the PR, it should make both of you happy :)